newIDE/app/src/ObjectsList/index.js

```diff
@@ -65,6 +65,7 @@ const objectTypeToDefaultName = {
   'BitmapText::BitmapTextObject': 'NewBitmapText',
   'TextEntryObject::TextEntry': 'NewTextEntry',
   'TileMap::TileMap': 'NewTileMap',
+  'TileMap::CollisionMask': 'NewTileMapMask',
```

Comment:
noob question: What is this for?

Reply:
It's when you create a new object, it gives a default name to it.